static_files/dora-config/config.yaml.tmpl

```diff
@@ -41,31 +37,27 @@ beaconapi:
       archive: true
 {{- end }}
   # local cache for page models
-  localCacheSize: 100 # 100MB
+  localCacheSize: 10 # 10MB
```

Comment:
not sure what this is used for but is reducing this alright?

Reply:
yea, the cache is used to preserve page models.
I've set it to a high limit to handle public testnet scenarios (so with probably hundreds of uses using the explorer at the same time)

but for a local only testnet 100MB is quite high, so I've reduced it to 10MB which is still enough to cache hundreds of page models :)
Lower cache does not really affect the explorer functionality. Page processing is quite fast, even with the localCache disabled at all (debug mode)